packages/rich-text-html-renderer/src/__test__/index.test.ts

```diff
@@ -205,14 +206,24 @@ describe('documentToHtmlString', () => {
   it('renders tables', () => {
     const document: Document = tableDoc;
     const expected =
-      '<table><tbody>' +
+      '<table>' +
```

Comment:
Why did we remove tbody? Because th cannot be rendered inside it?

Reply:
Yes it won't be semantically correct HTML